timemachine/fe/single_topology.py

```diff
@@ -166,6 +178,19 @@ def setup_dummy_interactions(
         if all([a in dga for a in idxs]):
             dummy_improper_idxs.append(tuple([int(x) for x in idxs]))
             dummy_improper_params.append(params)
+    for idxs, params in zip(chiral_atom_idxs, chiral_atom_params):
+        # consider a few cases: (1) all idxs dummy, (2) central atom anchor / rest dummy, (3) other
+
+        center, i, j, k = idxs
+
+        case_1 = all(a in dummy_group for a in (center, i, j, k))
+        case_2 = (center == root_anchor_atom) and all(a in dummy_group for a in (i, j, k))
+        # TODO: maybe add more cases, if we convince ourselves they are also factorizable...
+        #    e.g. if center is root_anchor_atom, and at most one of (i, j, k) is neighbor_anchor_atom
+        #   (and maybe also if root_anchor_atom, neighbor_anchor_atom are swapped in above statement)
+        if case_1 or case_2:
+            dummy_chiral_atom_idxs.append(tuple(int(x) for x in idxs))
+            dummy_chiral_atom_params.append(params)
```

Comment:
@proteneer : I pushed a couple changes to your branch based on offline discussion.

(This now covers a couple cases that seem like safe additions, but this should be double-checked.)

Reply:
I did some back of envelop math and I'm fairly certain now that if exactly one atom is a root_anchor atom, and everything else is a dummy atom in the grpup, then it's also fine. (Since it will never affect the inter atomic distances of the core atoms, as whatever force you apply is only to a single core anchor atom). I'll modify the PR to address this.